### ISSUE TITLE
Focus on the command end of Borland

### ISSUE THREAD

**Issue by mcdonnelldean** [continuationlabs/borland#11]
@geek Mentioned he wanted to focus on borland as a plugin not borland as a UI. I agree with this. I already have a UI to surface Borland's command interface too toolbag.

I need to be able to switch off the data collection ability but have the command ability available. Is there a list of commands that borland can send to the process outside of data capture, termination for instance? I could add these to the UI and hook in borland no problem.

<img width="1680" alt="screen shot 2016-02-08 at 20 25 55" src="https://cloud.githubusercontent.com/assets/3625961/12898056/9a05ffa8-cea2-11e5-9b14-2c2523566bc4.png">

We can also add the data processing now that I think of it. If I can configure what these do I can basically send the data seneca-stats to be moved around the system. This would enable me to dump the real time data after viewing and only store the snapshots to influx.

Basically, we need to all chat :D

**Comment by mcdonnelldean** [comment]
@mcollina This will interest you too, we can basically have real time but only store snapshots since Borland can send a snapshot via websocket on command. I can handle this, write it to seneca-stats and have it stored. 

I'm sure we could also add scheduling pretty easy, we just need trigger borland, which will trigger toolbag. I think a simple set of hapi plugins would give us what we need :D

**Comment by cjihrig** [comment]
> Is there a list of commands that borland can send to the process outside of data capture, termination for instance?

Yes. The list depends on the plugins that are registered. When borland is running, go to `/clients` to see the connected clients and the commands each one support.

> I need to be able to switch off the data collection ability but have the command ability available.

I think `reporter-stop` is the command you want. However, you could also just not use the stats-collector plugin, or configure it to not be running by default.

**Comment by mcdonnelldean** [comment]
@cjihrig Awesome let me plug borland in and check it out so. On the stats, I think I am going to leave them on because I need snapshots soon too.

**Comment by mcdonnelldean** [comment]
<img width="865" alt="screen shot 2016-02-12 at 16 16 06" src="https://cloud.githubusercontent.com/assets/3625961/13012697/3693dc88-d1a4-11e5-993c-aacc6cac7d74.png">

This is what it's looking like :D

**Comment by cjihrig** [comment]
The UI is shaping up so well!
